docs/getting_started/tutorial.md

```diff
@@ -155,30 +234,46 @@ class HomePage(Page):
     ]
 ```
 
-`body` is defined as `RichTextField`, a special Wagtail field. When `blank=True`,
-it means that this field is not required and can be empty. You
-can use any of the [Django core fields](https://docs.djangoproject.com/en/stable/ref/models/fields). `content_panels` define the
-capabilities and the layout of the editing interface. When you add fields to `content_panels`, it enables them to be edited on the Wagtail interface. [More on creating Page models](../topics/pages).
+In this code snippet above, `body` is defined as `RichTextField`, a special Wagtail field.
+When `blank=True`, this field is not required and can be empty.
 
-Run:
+You can use any of the [Django core fields](https://docs.djangoproject.com/en/stable/ref/models/fields). Notice that `content_panels` define the
+capabilities and the layout of the editing interface. When you add fields to `content_panels`,
+it enables these fields to be edited in the Wagtail interface. [More on creating `Page` models](../topics/pages).
+
+Before we can see the changes, we will need to perform a database migration. Simply explained, a database migration refers to the process of moving data from one database to another. In this case, our initial empty database to a new database with a `HomePage` model.
```

Comment:
This isn't really correct - in this case "database migration" refers to changing the structure of the database while keeping its data intact, not moving data between databases.